### ISSUE TITLE
Update Credits yml file template

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#2642]
### Overview
As a member of the team, we need to ensure the Wiki is an up to date and accurate resource. For this issue, we will update the [Credits yml file template](https://github.com/hackforla/website/wiki/Credits-yml-file-template).

### Action Items
This will be an issue worked on and discussed during development team meetings.
- [x] Figure out why there are two name values for (content and type) which seem the same
  - [x] You may want to see how (if anywhere) content and type are used in our code 
- [x] Update the [Credits yml file template](https://github.com/hackforla/website/wiki/Credits-yml-file-template)
- [x] Release dependency on https://github.com/hackforla/website/issues/2633
  - [x] Update action items in the dependency issue if needed 

### Resources/Instructions
- [Credits yml file template](https://github.com/hackforla/website/wiki/Credits-yml-file-template)
- [pages/credits.html](https://github.com/hackforla/website/blob/e33b5d42e8c048255b2448ca5567f511c85dc1c2/pages/credits.html)
- This issue might be helpful in figuring out what the difference between content and type are: https://github.com/hackforla/website/issues/571

**Comment by JessicaLucindaCheng** [comment]
All action items are completed, so I'm closing this issue.

**Comment by JessicaLucindaCheng** [comment]
Note about decisions made: For the [Credits yml file template](https://github.com/hackforla/website/wiki/Credits-yml-file-template), originally, it had both the `content` and `type` field, which seemed similar in terms of its use, and neither field was used in the code itself. Thus, we decided to remove the `type` field because only one of the fields is necessary if they contain the same information.  Also, we changed the `content` field to the `content-type` field to make the field name clearer for what its purpose is, which is type of content. In addition, we decided the `content-type` field should be standardized to only certain choices; for now, those are image, video, or audio but other types of content can be added in the future if necessary and should be added to the template so that every credit we create will use the same values for `content-type`. This will allow developers a way to differentiate easily between different content types in the future and display them correctly on the website if we start using different content types.
